docs/guide/general/architecture.md

```diff
@@ -0,0 +1,36 @@
+# Architecture
+
+_Vue Storefront Next_ has been built upon the idea of _microservice architecture_, that is, also known as, including but not limited to _agnostic design_, _declarative protocol_, _modular packages_, and basically all the good things you can think of when it comes to software development. (It's not the one shot perfect solution, but let's focus on good things first)
```

Comment:
Things that I would like to highlight in the architecture docs (not limited to)

- comes with a fully working example project as a default theme
- just set of npm packages
 - each package is independent
- within package each composable can by used on its own (for example you don't have anything else to use `useCart` `useProduct` etc)
- works everywhere (nuxt, vue-cli projects, custom vue projects, can be embedded literally everywhere)
- optional agnosticity
- uses composition api

Reply:
Duly noted, good points